README.md

```diff
@@ -14,7 +14,7 @@ Avatar Animator generates real-time 2D vector-based avatar animations using webc
 
 ![Gif animation showing a demo of Avatar Animator using Microsoft Teams](images/demo.gif)
 
-**Demo of Avatar Animator using Microsoft Teams**. Instead of using a real webcam as the input source to Avatar Animator, the input is simulated with [OBS Studio](https://obsproject.com/) and [OBS-VirtualCam](https://github.com/Fenrirthviti/obs-virtual-cam) from a [YouTube video](https://www.youtube.com/watch?v=x_lHk9Lf-ow), leading to the following processing chain: YouTube (browser) 🠚 OBS Studio (desktop capture) 🠚 OBS-VirtualCam (1) 🠚 Avatar Animator 🠚 OBS-VirtualCam (2) 🠚 Microsoft Teams. Why? Just for fun...
+**Figure 1**: Demo of Avatar Animator using Microsoft Teams. Instead of using a real webcam as the input source to Avatar Animator, the input is simulated with [OBS Studio](https://obsproject.com/) and [OBS-VirtualCam](https://github.com/Fenrirthviti/obs-virtual-cam) from a [YouTube video](https://www.youtube.com/watch?v=x_lHk9Lf-ow), leading to the following processing chain: YouTube (browser) 🠚 OBS Studio (desktop capture) 🠚 OBS-VirtualCam (1) 🠚 Avatar Animator 🠚 OBS-VirtualCam (2) 🠚 Microsoft Teams. Why? Just for fun...
```

Comment:
I'm not a fan of using "Figure xy" in these docs. I looks too much like a paper. I would leave it as it was.

Reply:
I did it for clarity as I preferred it rather than something like "figure below left" -- nothing to do with sounding like a paper. In any case I reverted this in https://github.com/letmaik/avatar-animator/pull/29/commits/83eaeefbd3e6767c3005712d1c06d49abe5b3e15